FinniversKit/Sources/Components/Label/Label.swift

```diff
@@ -15,7 +15,7 @@ public class Label: UILabel {
 
     public init(
         style: Style,
-        numberOfLines: Int = 1,
+        numberOfLines: Int = 2,
```

Comment:
This default value should not be changed, since it'll break other usages. Instead, pass `numberOfLines: 2` when instantiating the labels you need to have 2 lines.